sqlglot/parser.py

```diff
@@ -1423,6 +1432,11 @@ def _parse_describe(self) -> exp.Expression:
 
         return self.expression(exp.Describe, this=this, kind=kind)
 
+    def _parse_or_cond(self):
```

Comment:
add type -> TokenType

Reply:
I can't figure out what I'm supposed to return here